src/ServiceControl/Infrastructure/TransportMessageExtensions.cs

```diff
@@ -16,18 +16,20 @@ public static string ProcessingEndpointName(this IReadOnlyDictionary<string, str
                 return endpoint;
             }
 
+            // This message could be a failed message.
+            if (headers.TryGetValue("NServiceBus.FailedQ", out endpoint))
+            {
+                return endpoint;
+            }
+
+            // In v5, a message that comes through the Audit Queue
+            // has it's ReplyToAddress overwritten to match the processing endpoint
             var replyToAddress = headers.ReplyToAddress();
```

Comment:
@ramonsmits At the moment we'll only use ReplyToAddress if FailedQ is not present. All failed messages should have a FailedQ header. Should we still check for another header?